src/components/dashboard/Dashboard.js

```diff
@@ -865,37 +864,58 @@ const Dashboard = props => {
                         plain
                       />
                     </TouchableOpacity>
-                    {/* {walletChatEnabled && (
-                      <WalletChatWidget
-                        connectedWallet={
-                          web3Provider
-                            ? {
-                                walletName: 'GoodWalletV2',
-                                account: goodWallet.account,
-                                chainId: goodWallet.networkId,
-                                provider: web3Provider, //goodWallet.wallet.currentProvider
-                              }
-                            : undefined
-                        }
-                      />
-                    )} */}
                   </Animated.View>
-                  {headerLarge && (
-                    <Animated.View style={[styles.headerFullName, fullNameAnimateStyles]}>
-                      <Section.Text color="gray100Percent" fontFamily={theme.fonts.default} fontSize={12}>
-                        {fullName || ' '}
-                      </Section.Text>
-                    </Animated.View>
-                  )}
+                    <WalletChatWidget
+                      connectedWallet={
+                        web3Provider
```

Comment:
1. remove comment
2. check does it works (updates) properly when you switch chain

Reply:
Thanks for the review, i have updated the PR